com.unity.sg2/Editor/GraphDeltaRegistry/GraphDelta/GraphHandler.cs

```diff
@@ -209,7 +254,7 @@ private void AddEntry(NodeHandler context, CPEntryDescriptor desc)
             string cpName,
             bool input)
         {
-            
```

Comment:
All of the above is just to make it so concretization happens topologically and without propagation.

Normally, ReconcretizeNode sends a concretization impulse to downstream nodes. ReconcretizeAll would then reconcretize ALL nodes (thus triggering downstream impulses also), resulting in nodes concretizing multiple times. Since Dynamic vector resolution requires the incoming connections to be from concretized nodes, this would also means the concretizations could happen in the wrong order.